playground/backend/internal/fs_tool/lc_constructor.go

```diff
@@ -30,35 +30,51 @@ func newCompilingLifeCycle(pipelineId uuid.UUID, pipelinesFolder, sourceFileExte
 	baseFileFolder := filepath.Join(pipelinesFolder, pipelineId.String())
 	srcFileFolder := filepath.Join(baseFileFolder, sourceFolderName)
 	binFileFolder := filepath.Join(baseFileFolder, compiledFolderName)
+
+	srcFileName := pipelineId.String() + sourceFileExtension
+	absSrcFileFolderPath, _ := filepath.Abs(srcFileFolder)
+	absSrcFilePath, _ := filepath.Abs(filepath.Join(absSrcFileFolderPath, srcFileName))
```

Comment:
Don't you need to process errors somehow?

Reply:
I guess no. `filepath.Abs()` returns `err` only if `os.Getwd()` returns `err`.